src/common/util/useCurrentPerson.ts

```diff
@@ -39,4 +40,25 @@ export function updateCurrentPerson() {
   }
 }
 
+export function updateCurrentPersonPassword() {
+  const { data: session } = useSession()
+  return async (data: Credentials) => {
+    return await apiClient.put<Credentials, AxiosResponse<boolean>>(
+      endpoints.account.updatePassword.url,
+      data,
+      authConfig(session?.accessToken),
+    )
+  }
+}
+
+export function disableCurrentPerson() {
+  const { data: session } = useSession()
+  return async () => {
+    return await apiClient.delete<UpdatePerson, AxiosResponse<Person>>(
+      endpoints.account.update.url,
+      authConfig(session?.accessToken),
+    )
+  }
+}
```

Comment:
Do we need this function? It will be strange for someone to want to disable his own account. A more likely action would be for an Admin to disable someone else's account - so the methog might be needed but with personId as parameter.